features/bicycle/names.feature

```diff
@@ -16,7 +16,7 @@ Feature: Bike - Street names in instructions
 
         When I route I should get
             | from | to | route                                   |
-            | a    | c  | My Way (A6),Your Way (A7),Your Way (A7) |
+            | a    | c  | My Way,Your Way,Your Way                |
```

Comment:
these have to be added as an additional check instead of being removed `| route | ref |` ... or something like this. References are an essential part of guidance checks.